spec.html

```diff
@@ -31640,14 +31640,17 @@ <h1>Notation</h1>
           <li>
             _Unicode_ is *true* if the RegExp object's [[OriginalFlags]] internal slot contains *"u"* and otherwise is *false*.
           </li>
+          <li oldids="sec-runtime-semantics-wordcharacters-abstract-operation">
+            _WordCharacters_ is the mathematical set that is the union of all sixty-three characters in *"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_"* (letters, numbers, and U+005F (LOW LINE) in the Unicode Basic Latin block) and all characters _c_ for which _c_ is not in that set but Canonicalize(_c_) is. _WordCharacters_ cannot contain more than sixty-three characters unless _Unicode_ and _IgnoreCase_ are both *true*.
```

Comment:
> `_WordCharacters_` cannot contain more than sixty-three characters unless `_Unicode_` and `_IgnoreCase_` are both `*true*`.

This scares me a little because it could become false in with a new version of Unicode, and we incorporate the latest version by reference. (Unless they've committed to not doing that, which would be great.)

But I guess it's fine.

Reply:
It's dependent only upon the [Canonicalize](https://tc39.es/ecma262/#sec-runtime-semantics-canonicalize-ch) operation continuing to use those same conditions as a guard for projecting its input into a single ASCII character. We're safe unless Unicode toUppercase starts mapping ASCII punctuation into letters, but I'll still try to clarify a bit.

Reply:
> We're safe unless Unicode toUppercase starts mapping ASCII punctuation into letters

Yeah, fair enough, that's not worth worrying about.